Wave1_data/1.preprocess_data/scripts/0.preprocess_raw_data.py

```diff
@@ -0,0 +1,580 @@
+    "0252": "K12",
+    "0253": "K13",
+    "0254": "K14",
+    "0255": "K15",
+    "0256": "K16",
+    "0257": "K17",
+    "0258": "K18",
+    "0259": "K19",
+    "0260": "K20",
+    "0261": "K21",
+    "0262": "K22",
+    "0263": "K23",
+    "0264": "K24",
+    "0265": "L01",
+    "0266": "L02",
+    "0267": "L03",
+    "0268": "L04",
+    "0269": "L05",
+    "0270": "L06",
+    "0271": "L07",
+    "0272": "L08",
+    "0273": "L09",
+    "0274": "L10",
+    "0275": "L11",
+    "0276": "L12",
+    "0277": "L13",
+    "0278": "L14",
+    "0279": "L15",
+    "0280": "L16",
+    "0281": "L17",
+    "0282": "L18",
+    "0283": "L19",
+    "0284": "L20",
+    "0285": "L21",
+    "0286": "L22",
+    "0287": "L23",
+    "0288": "L24",
+    "0289": "M01",
+    "0290": "M02",
+    "0291": "M03",
+    "0292": "M04",
+    "0293": "M05",
+    "0294": "M06",
+    "0295": "M07",
+    "0296": "M08",
+    "0297": "M09",
+    "0298": "M10",
+    "0299": "M11",
+    "0300": "M12",
+    "0301": "M13",
+    "0302": "M14",
+    "0303": "M15",
+    "0304": "M16",
+    "0305": "M17",
+    "0306": "M18",
+    "0307": "M19",
+    "0308": "M20",
+    "0309": "M21",
+    "0310": "M22",
+    "0311": "M23",
+    "0312": "M24",
+    "0313": "N01",
+    "0314": "N02",
+    "0315": "N03",
+    "0316": "N04",
+    "0317": "N05",
+    "0318": "N06",
+    "0319": "N07",
+    "0320": "N08",
+    "0321": "N09",
+    "0322": "N10",
+    "0323": "N11",
+    "0324": "N12",
+    "0325": "N13",
+    "0326": "N14",
+    "0327": "N15",
+    "0328": "N16",
+    "0329": "N17",
+    "0330": "N18",
+    "0331": "N19",
+    "0332": "N20",
+    "0333": "N21",
+    "0334": "N22",
+    "0335": "N23",
+    "0336": "N24",
+    "0337": "O01",
+    "0338": "O02",
+    "0339": "O03",
+    "0340": "O04",
+    "0341": "O05",
+    "0342": "O06",
+    "0343": "O07",
+    "0344": "O08",
+    "0345": "O09",
+    "0346": "O10",
+    "0347": "O11",
+    "0348": "O12",
+    "0349": "O13",
+    "0350": "O14",
+    "0351": "O15",
+    "0352": "O16",
+    "0353": "O17",
+    "0354": "O18",
+    "0355": "O19",
+    "0356": "O20",
+    "0357": "O21",
+    "0358": "O22",
+    "0359": "O23",
+    "0360": "O24",
+    "0361": "P01",
+    "0362": "P02",
+    "0363": "P03",
+    "0364": "P04",
+    "0365": "P05",
+    "0366": "P06",
+    "0367": "P07",
+    "0368": "P08",
+    "0369": "P09",
+    "0370": "P10",
+    "0371": "P11",
+    "0372": "P12",
+    "0373": "P13",
+    "0374": "P14",
+    "0375": "P15",
+    "0376": "P16",
+    "0377": "P17",
+    "0378": "P18",
+    "0379": "P19",
+    "0380": "P20",
+    "0381": "P21",
+    "0382": "P22",
+    "0383": "P23",
+    "0384": "P24",
+}
+# write the well map to a json file
+path_to_repo_data = pathlib.Path("../../../data/processed/").resolve()
+path_to_repo_data.mkdir(exist_ok=True, parents=True)
+with open(path_to_repo_data / "well_map.json", "w") as f:
+    json.dump(well_map, f)
+# map the well to the well_map
+df["Well"] = df["Well"].map(well_map)
+df.head()
```

Comment:
Same as comment above, is this json file meant to be the platemap file or what makes this different?

Reply:
The json is for mapping the well number to an actual well name